[2022-06-12 16:04:11,793] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ingest_data.save_data_task scheduled__2022-06-11T21:48:00+00:00 [queued]>
[2022-06-12 16:04:11,821] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ingest_data.save_data_task scheduled__2022-06-11T21:48:00+00:00 [queued]>
[2022-06-12 16:04:11,822] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-12 16:04:11,823] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-12 16:04:11,823] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-12 16:04:11,883] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): save_data_task> on 2022-06-11 21:48:00+00:00
[2022-06-12 16:04:11,890] {standard_task_runner.py:52} INFO - Started process 1568 to run task
[2022-06-12 16:04:11,895] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'ingest_data', 'save_data_task', 'scheduled__2022-06-11T21:48:00+00:00', '--job-id', '131', '--raw', '--subdir', 'DAGS_FOLDER/ingestion_dag.py', '--cfg-path', '/tmp/tmpkdfx3x_3', '--error-file', '/tmp/tmpq5hrpy0h']
[2022-06-12 16:04:11,897] {standard_task_runner.py:80} INFO - Job 131: Subtask save_data_task
[2022-06-12 16:04:12,034] {task_command.py:370} INFO - Running <TaskInstance: ingest_data.save_data_task scheduled__2022-06-11T21:48:00+00:00 [running]> on host DESKTOP-GDPK8BL.localdomain
[2022-06-12 16:04:12,421] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=ingest_data
AIRFLOW_CTX_TASK_ID=save_data_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-11T21:48:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-11T21:48:00+00:00
[2022-06-12 16:04:12,437] {python.py:173} INFO - Done. Returned value was: None
[2022-06-12 16:04:12,462] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=ingest_data, task_id=save_data_task, execution_date=20220611T214800, start_date=20220612T140411, end_date=20220612T140412
[2022-06-12 16:04:12,552] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-12 16:04:12,792] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
